assert/string.go

```diff
@@ -0,0 +1,37 @@
+package assert
+
+import (
+	"regexp"
+	"strings"
+	"testing"
+)
+
+// StringContains asserts that string s contains the subst string and calls
+// the failure function with details otherwise.
+func (assert *Assert) StringContains(s string, substr string, details ...interface{}) {
+	assert.IsTrue(strings.Contains(s, substr), "strings.Contains(%q, %q).%s",
```

Comment:
This is mostly cosmetic but I personally prefer this to be just like how we chain errors:

```suggestion
	assert.IsTrue(strings.Contains(s, substr), "strings.Contains(%q, %q): %s",
```

The single dot reads funny to me, but again, not necessarily bad/extremely cosmetical/feel free to disregard.